models/src/main/java/com/vimeo/networking2/PublishToFacebookPost.kt

```diff
@@ -0,0 +1,61 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Represents the required data for a Facebook post.
+ */
+@JsonClass(generateAdapter = true)
+data class PublishToFacebookPost(
+
+        /**
+         * The title of the post as it will appear on Facebook.
+         */
+        @Json(name = "title")
+        val title: String,
```

Comment:
This and all properties in this class should be nullable with a default value of null. This comment also applies to
- `PublishToLinkedInPost`
- `PublishToTwitterPost`
- `PublishToYouTubePost`

Unless the API ignores field filters for these non null properties they should be made null.